components/new-contribution-flow/ContributionFlowMainContainer.js

```diff
@@ -163,14 +172,16 @@ class NewContributionFlowMainContainer extends React.Component {
     const { LoggedInUser, step } = this.props;
 
     return (
-      <StyledCard p={32} borderRadius={15}>
+      <StyledCard p={[16, 32]} mx={[16, 'none']} borderRadius={15}>
         <Flex flexDirection="column" alignItems="center">
           <Flex width="100%" mb={3}>
-            <H4 fontWeight={500} py={2}>
-              {this.renderHeader(step.name, LoggedInUser)}
-            </H4>
+            <Flex width={3 / 4} alignItems="center" justifyContent="center">
+              <H4 fontSize={[20, 24]} fontWeight={500} py={2}>
+                {this.renderHeader(step.name, LoggedInUser)}
+              </H4>
+            </Flex>
             <Flex flexGrow={1} alignItems="center" justifyContent="center">
-              <StyledHr width="100%" ml={3} borderColor="black.300" />
+              <StyledHr width="100%" ml={['none', 3]} borderColor="black.300" />
```

Comment:
Here you can just do `[null, 3]`, `StyledHr` has no margin left by default so you don't need to output any style.

```suggestion
              <StyledHr width="100%" ml={[null, 3]} borderColor="black.300" />
```